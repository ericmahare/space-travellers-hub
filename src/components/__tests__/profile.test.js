import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import MyProfile from '../components/profile/profile';

const ProfileProvider = () => (
  <Provider store={store}>
    <MyProfile />
  </Provider>
);

describe('test profile component', () => {
  it('render profile correctly', () => {
    const myProfile = renderer.create(<ProfileProvider />);
    expect(myProfile).toMatchSnapshot();
  });
});