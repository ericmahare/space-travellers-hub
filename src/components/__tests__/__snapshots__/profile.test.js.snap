// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test profile component render profile correctly 1`] = `
<div
  className="profile"
>
  <div
    className="box"
  >
    <h2>
      My Reserved Rockets
    </h2>
    <table
      className="Mission-ProfileTable"
    >
      <tbody />
    </table>
  </div>
  <div
    className="box"
  >
  <div>
    <div
      className="box"
    >
      <h2>
        My Rockets
      </h2>
      <table
        className="Mission-ProfileTable"
      >
        <tbody />
      </table>
    </div>
  </div>
</div>
`;
